src/extensions/Seo/Seo.js

```diff
@@ -187,10 +198,17 @@ const Seo = ({ sdk }) => {
           labelText="Canonical URL"
           helpText="Canonical specifies to search engines your preferred URL. Default is current page URL."
           textInputProps={{
-            maxLength: 60,
+            maxLength: 256,
             onKeyPress: (e) => onFieldChangeDebounce(e.currentTarget),
             onBlur: (e) => onFieldChange(e.currentTarget),
-            testId: 'Seo-tabpanel-general-canonical'
+            testId: 'Seo-tabpanel-general-canonical',
+            onChange: (e) => {
+              if (e.currentTarget?.value?.length > 60) {
```

Comment:
Go ahead and remove this for the canonical URL URLs can be long, I had thought this was for raising the page title length. Save the function and add the warning to the page title field.